src/application-base/private/getBrowserLocale.js

```diff
@@ -24,6 +24,13 @@ const filterLanguages = () => {
   if (navigator.languages && navigator.languages.length > 0) {
     for (let index = 0; index < navigator.languages.length; index += 1) {
       const locale = navigator.languages[index];
+      const fallbackLocale = locale.split('-').length > 1 ? locale.split('-')[0] : false;
```

Comment:
We should avoid mix and matching types when possible. In this case the fallbackLocale can be a `string` or a `boolean`. The exceptions are `undefined` and `null`. This assignment should resolve to a string or `undefined`/`null`.

Reply:
changed here 7731a62c36827a92ca385599cec577dc79d6fd1e